quickjs.c

```diff
@@ -7007,6 +7007,11 @@ static JSValue JS_GetPrototypeFree(JSContext *ctx, JSValue obj)
     return obj1;
 }
 
+/* works for arrays, strings, functions and arguments, but not for typed arrays */
+int JS_ArrayGetLength(JSContext *ctx, int64_t *pres, JSValueConst obj) {
+    return js_get_length64(ctx, pres, obj);
+}
+
```

Comment:
It is misleading for this function to **not** handle typed arrays given the name.
The name is even more confusing as it does handle strings and functions.